narwhal/consensus/src/bullshark.rs

```diff
@@ -194,22 +194,34 @@ impl ConsensusProtocol for Bullshark {
             .with_label_values(&["elected"])
             .inc();
 
+        self.metrics
```

Comment:
Maybe leave a comment on the code that these metrics will not be consistent between validators, to make sure someone does not in the future use them to derive reputation.

Reply:
Added.